client/signup/steps/user/index.jsx

```diff
@@ -458,7 +458,22 @@ export class UserStep extends Component {
 	}
 
 	getHeaderText() {
-		const { flowName, oauth2Client, translate, headerText, wccomFrom, isSocialFirst } = this.props;
+		const {
+			flowName,
+			oauth2Client,
+			translate,
+			headerText,
+			wccomFrom,
+			isSocialFirst,
+			userLoggedIn,
+			locale,
+		} = this.props;
+
+		if ( userLoggedIn ) {
+			if ( englishLocales.includes( locale ) ) {
+				return translate( 'Is that you?' );
```

Comment:
Not sure if the copy is final, but I'd suggest that "Is this you?" would make more sense given the UI here.

Reply:
Yes, makes total sense, I just updated the copy 👍